js/app/packages/app/component/mobile/MobileDock.css

```
@@ -0,0 +1,34 @@
+@keyframes morePopoverShow {
```

Stylelint (17.13.0)

[error] Expected keyframe name "morePopoverShow" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)